src/main/java/stsjorbsmod/cards/CustomJorbsModCard.java

```diff
@@ -238,4 +240,11 @@ public void triggerOnGlowCheck() {
         }
         return null;
     }
+
+    @Override
+    public void receiveStartAct() {
+        if (this.exert) {
```

Comment:
I don't think this is going to do the right thing with `Shrapnel Bloom`, where it wants to make cards exert as a one-time thing without necessarily being permanently exert-on-play. I think probably it would make more sense to have separate fields for `exerted` vs `selfExert`/`exert`